src/core.js

```diff
@@ -86,8 +86,6 @@ extend( QUnit, {
 
 	only: only,
 
-	// DEPRECATED: The functionality of QUnit.start() will be altered in QUnit 2.0.
-	// In QUnit 2.0, invoking it will ONLY affect the `QUnit.config.autostart` blocking behavior.
```

Comment:
nope, actually the current code is dependent on the whole method. Until I do a coverage check or minimize the complexity, I'm leaving it as it is. It won't have `QUnit.stop` to work along, at least.